tests/integration/local/start_api/test_start_api_cdk_template.py

```diff
@@ -0,0 +1,83 @@
+from pathlib import Path
+
+import pytest
+import requests
+from parameterized import parameterized_class
+
+from tests.integration.local.start_api.start_api_integ_base import StartApiIntegBaseClass
+
+
+@parameterized_class(
+    ("template",),
+    [
+        (Path("/testdata/start_api/cdk/template-rest-api.yaml"),),
+        (Path("/testdata/start_api/cdk/template-open-api.yaml"),),
+        (Path("/testdata/start_api/cdk/template-http-api.yaml"),),
+    ],
+)
+class TestStartAPICDKTemplateRestAPI(StartApiIntegBaseClass):
+    template_path = "/testdata/start_api/cdk/template-rest-api.yaml"
```

Comment:
Suppose it is `template_path` in `@parameterized_class`, do we to override it here?

Reply:
Good catch. Just forgot to remove this line when I was cleaning up.